docs/Microservices/RPC/buildRPC_Manual.md

```diff
@@ -0,0 +1,228 @@
+    ``` bash
+    vcpkg install cpprestsdk[websockets]:x64-windows-static
+    ```
+
+    **Build RPC**
+
+    1. Return to the `rpc` directory and create a new `build` directory.
+    ``` bash
+    cd .. && mkdir build && cd build
+    ```
+    
+    2. Generate the CMake config.
+    ``` bash
+    cmake -DVCPKG_TARGET_TRIPLET=x64-windows-static -DCMAKE_TOOLCHAIN_FILE=../vcpkg/scripts/buildsystems/vcpkg.cmake ..
+    ```
+
+    3. Build the RPC executable.
+    ```bash
+    cmake --build . --config Release
+    ```
+    
+        !!! note
+            RPC can also be built in a non-production debug mode rather than release using the following command. The debug mode includes debug symbols.
+            ```
+            cmake --build . --config Debug
+            ```
+
+    4. Change to `Release` directory.
+    ``` bash
+    cd Release
+    ```
+
+
+=== "Ubuntu/CentOS8"
+    The following steps are for Ubuntu 18.04, Ubuntu 20.04, or CentOS8.
+
+    **Build VCPKG and C++ REST SDK**
+
+    1. To install the required dependencies; enter the following command.
+
+        === "Ubuntu"
+        ``` bash
+        sudo apt install git cmake build-essential curl zip unzip tar pkg-config
+        ```
+    
+        === "CentOS8"
+        ``` bash
+        sudo yum install cmake
+        ```
+
+    2. In the `rpc` directory, clone the Vcpkg repository. Vcpkg is a C/C++ Library Manager for Windows that was created by Microsoft.  Find out more about it [here](https://github.com/microsoft/vcpkg).
+    ``` bash
+    git clone --branch 2020.11-1 https://github.com/microsoft/vcpkg.git && cd vcpkg
+    ```
+
+    3. Build vcpkg.exe using the following command.
+    ``` bash
+    ./bootstrap-vcpkg.sh
+    ```
+        
+    4. Install C++ REST SDK. This can take anywhere from 8 - 15 minutes depending on download speeds and installation times.
+    ``` bash
+    ./vcpkg install cpprestsdk[websockets]
+    ```
+
+    **Build RPC**
+
+    1. Return to the `rpc` directory and create a new 'build' directory.
+    ``` bash
+    cd .. && mkdir build && cd build
+    ```
+    
+    2. Generate the CMake config.
+    ``` bash
+    cmake -DCMAKE_TOOLCHAIN_FILE=../vcpkg/scripts/buildsystems/vcpkg.cmake -DCMAKE_BUILD_TYPE=Release ..
+    ```
+
+        !!! note
+            RPC can also be built in a non-production debug mode rather than release using the following command. The debug mode includes debug symbols.
+            ```
+            cmake -DCMAKE_TOOLCHAIN_FILE=../vcpkg/scripts/buildsystems/vcpkg.cmake -DCMAKE_BUILD_TYPE=Debug ..
+            ```
+
+    3. Build the RPC executable.
+    ```bash
+    cmake --build .
+    ```
+
+=== "CentOS7"
+    !!! important
+        **The "export PATH=..." (for CMake and Git), and "scl enable devtoolset-7 bash" (for GCC) must be executed in in the Terminal you are building from; i.e. these are temporary changes which only affect the current Terminal session.**
+
+    **Install Dependencies**
+
+    1. Download CMake.
```

Comment:
Do we want to point users to where they can download it from?

Reply:
Sounds like a good idea to add the link.